Header header  # same as in point_cloud

int32 class_id
int32 tracking_id
sensor_msgs/PointCloud2 point_cloud

